sample/MinimalSample/MinimalSample.csproj

```diff
@@ -1,21 +1,19 @@
 <Project Sdk="Microsoft.NET.Sdk.Worker">
 
     <PropertyGroup>
-        <TargetFramework>net8.0</TargetFramework>
+        <TargetFramework>net9.0</TargetFramework>
```

Comment:
Shouldn't we stick with the 8.0 LTS for the samples?

Reply:
Hmmm good question. I thought always using latest and greatest as they might break easier than the LTS

Reply:
Let me roll it back. I also added Central Package Management.

Reply:
Fair enough